doc/Table.md

```diff
@@ -0,0 +1,281 @@
+| kmax           | int(11) | The number of phase space angles 
+| boinc_work     | text    | The absolute path of work directory 
+| boinc_results  | text    | The absolute path of results directory 
+| surv_percent   | int(11) | When testing jobs on HTCondor, survival percent decides whether to submit jobs to boinc
+
+## templates
+| Field     | Type | Description 
+|-----------|------|------------
+| mask_file | blob | The mask file of madx input file 
+| fort_3    | blob | The mother file of fort.3 
+
+## preprocess_wu
+| Field      | Type       | Description 
+|------------|------------|------
+| wu_id      | int(11)    | The work unit id
+| job_name   | text       | The job name  
+| input_file | blob       | The input file for preprocess.py  
+| batch_name | text       | The batch name of this job in HTCondor
+| unique_id  | text       | The unique id of this job on HTCondor, usually is ClusterId.ProcId  
+| status     | text       | The status of this job, 'complete', 'incomplete' or 'submitted'  
+| task_id    | int(11)    | The task id which point to a submission  
+| mtime      | bigint(20) | The Last modification time  
+| SEEDRAN    | int(11)    | The seed for madx job   
+| QP         | int(11)    | A parameter for the machine  
+| IOCT       | int(11)    | A parameter for the machine  
+
+## preprocess_task
+| Field       | Type       | Description 
+|-------------|------------|------
+| task_id     | int(11)    | The unique task id
+| wu_id       | int(11)    | The work unit id
+| madx_in     | blob       | The input file of madx job
+| madx_stdout | blob       | The output file of madx job
+| job_stdout  | blob       | The standard output of HTCondor job 
+| job_stderr  | blob       | The standard err of HTCondor job 
+| job_stdlog  | blob       | The log file of HTCondor job 
+| status      | text       | The status of this task, 'Success' or 'Failed' 
+| mtime       | bigint(20) | Last modification time
+| fort_2      | mediumblob | fort.2
+| fort_3_mad  | mediumblob | fort.3.mad
+| fort_3_aux  | mediumblob | fort.3.aux
+| fort_8      | mediumblob | fort.8
+| fort_16     | mediumblob | fort.16
+| fort_34     | mediumblob | fort.34
+
+## oneturn_sixtrack_wu
+| Field        | Type    | Description 
+|--------------|---------|------
+| turnss       | int(11) | The tracking turn
+| nss          | int(11) | Amplitude step in beam delta 
+| ax0s         | double  | Start amplitude 
+| ax1s         | double  | End amplitude 
+| imc          | int(11) | Number of variations of the relative momentum deviation 
+| iclo6        | int(11) | This switch allows to calculate the 6D closed orbit and optical functions at the starting point, using the differential algebra package.
+| writebins    | int(11) | This defines after how many turns data are written to output files 
+| ratios       | int(11) | Denotes the emittance ratio (eII/eI) of horizontal and vertical motion 
+| Runnam       | text    | The running name 
```

Comment:
Without the `track` dir, this field denotes only the title of the job reported in `fort.3`